components/LikeNotification/LikeNotification.module.css

```diff
@@ -1,9 +1,11 @@
 .like-notification {
-    display: flex;
-    align-items: center;
-    justify-content: space-between;
 }
 
 .like-notification--unread {
-    background-color: var(--lightGray);
-}
\ No newline at end of file
+  background-color: var(--lightGray);
+}
+
```

Comment:
NO need to write this style, there is a 'date' class for this in global.scss file